### PR TITLE
feat(nutanix.support): add support tile component

### PR DESCRIPTION
**Pull request by varun257**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/nutanix`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-7557, #MANAGER-7594
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

Add support tile to nutanix node dashboard

### CODE REVIEW

File: packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -13,5 +13,11 @@
   "server_datacenter_VIN": "Vint Hill (VIN {{number}}) - USA",
   "server_datacenter_WAW": "Warschau (WAW {{number}}) - Polen",
   "server_datacenter_RBX_HZ": "Roubaix (RBXHZ) - Frankreich",
-  "server_datacenter_GSW": "(GSW) - Frankreich"
+  "server_datacenter_GSW": "(GSW) - Frankreich",
+  "nutanix_node_general_info_support_title": "Support",
+  "nutanix_node_general_info_support_level_title": "Support level",
+  "nutanix_node_general_info_support_ticket_title": "Tickets ouverts",
+  "nutanix_node_general_info_support_create_ticket": "Creer un ticket",
+  "nutanix_node_general_info_support_issue_question": "Un probleme sur vos produits?",
```
Comment (antleblanc) [inline packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/translations/Messages_fr_FR.json:21]:
```suggestion
  "nutanix_node_general_info_support_issue_question": "Un problème sur vos produits ?",
```

File: packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -13,5 +13,11 @@
...
+  "nutanix_node_general_info_support_issue_link_text": "Consulter nos taches travaux"
```
Comment (antleblanc) [inline packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/translations/Messages_fr_FR.json:22]:
```suggestion
  "nutanix_node_general_info_support_issue_link_text": "Consulter nos tâches travaux"
```

File: packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -13,5 +13,11 @@
   "server_datacenter_VIN": "Vint Hill (VIN {{number}}) - USA",
   "server_datacenter_WAW": "Warschau (WAW {{number}}) - Polen",
   "server_datacenter_RBX_HZ": "Roubaix (RBXHZ) - Frankreich",
-  "server_datacenter_GSW": "(GSW) - Frankreich"
+  "server_datacenter_GSW": "(GSW) - Frankreich",
+  "nutanix_node_general_info_support_title": "Support",
+  "nutanix_node_general_info_support_level_title": "Support level",
```
Comment (antleblanc) [inline packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/translations/Messages_fr_FR.json:18]: **question:** Shouldn't be:

```suggestion
  "nutanix_node_general_info_support_level_title": "Niveau de support",
```

Instead?

**Review by antleblanc — CHANGES_REQUESTED** [review]

**Review by antleblanc — APPROVED** [review]

**Comment by antleblanc** [comment]
Just noticed that "Some checks were not successful". Could you please have a look?
Thank you!

**Review by radireddy — APPROVED** [review]

**Pull request merged**